crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -231,6 +231,6 @@ sp_api::decl_runtime_apis! {
         fn extract_root_blocks(ext: &Block::Extrinsic) -> Option<Vec<RootBlock>>;
 
         /// Extract block object mapping for a given block
-        fn extract_block_object_mapping(block: Block) -> BlockObjectMapping;
+        fn extract_block_object_mapping(block: Block, successful_calls: Vec<Block::Hash>) -> BlockObjectMapping;
```

Comment:
Relatively minor, but I don't feel like `Block::Hash` is semantically correct here. We are not dealing with block hashes in this case, hence the type should be separate, even if it ends up being identical to the block header hash algorithm.

Reply:
fair! I was being lazy to find that type i guess